projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -67,13 +67,19 @@ public function register_routes() {
 							'description'          => 'Marks a checklist as dismissed by the user',
 							'type'                 => 'object',
 							'properties'           => array(
-								'slug'         => array(
+								'slug'            => array(
 									'description' => 'Checklist slug',
 									'type'        => 'string',
 									'enum'        => $this->get_checklist_slug_enums(),
 								),
-								'is_dismissed' => array(
-									'type' => 'boolean',
+								'is_dismissed'    => array(
+									'type'     => 'boolean',
+									'required' => false,
+									'default'  => false,
+								),
+								'dismissed_until' => array(
+									'description' => 'Timestamp of when the checklist should be shown again',
```

Comment:
Just noting that if you decide on https://github.com/Automattic/wp-calypso/pull/86010  that you'll be sending relative time lapses then this should be changed

Reply:
Update to use relative values 
[Use relative dismiss values](https://github.com/Automattic/jetpack/pull/34889/commits/0366b10171fc4ea0683b6f4cf1171a97c52441ac)